sympy/functions/special/delta_functions.py

```diff
@@ -31,6 +49,22 @@ class DiracDelta(Function):
 
     5) ``DiracDelta(x,k) = 0``, for all ``x != 0``
 
+    Examples
+    ========
+
+    >>> from sympy import DiracDelta, diff
+    >>> from sympy.abc import x
+
+    >>> diff(DiracDelta(x))
+    DiracDelta(x, 1)
+    >>> DiracDelta(x**2 - 1).fdiff()
+    DiracDelta(x**2 - 1, 1)
+    >>> diff(DiracDelta(x - 1),x,2)
+    DiracDelta(x - 1, 2)
+    >>> diff(DiracDelta(x**2 - 1),x,2)
+    2*(2*x**2*DiracDelta(x**2 - 1, 2) + DiracDelta(x**2 - 1, 1))
+
+
```

Comment:
This is the docstring for the ``DiracDelta`` class, you should show some more features of ``DiracDelta`` rather than just the differentiation.